libs/utils/src/sentry_init.rs

```diff
@@ -0,0 +1,23 @@
+use sentry::ClientInitGuard;
+
+pub fn init_sentry(
+    maybe_sentry_url: Option<&String>,
+    process_name: &str,
+) -> Option<ClientInitGuard> {
+    return match maybe_sentry_url {
+        Some(sentry_url) => {
+            let _guard = sentry::init((
+                sentry_url.as_str().to_string(),
```

Comment:
I think you should be able to take `Option<&str>` here and pass `&str` without any conversions. If not, I'd recommend taking `Option<String>` right away instead of cloning.

Reply:
Option<&str> didn't work because get_one returns Option<&String> (and you can't do str because get_one requires the type parameter to be clonable). Also Option<String> didn't work because you are cloning anyways just on the caller side.

The change that did work is: sentry_url.as_str().to_string() -> sentry_url.to_string(). There was no need to cast it as a &str before instantiating it as String. Not sure how to avoid cloning.

Reply:
Just `.as_str()` works for me: [`init`](https://docs.rs/sentry/latest/sentry/fn.init.html) takes `Into<ClientOptions>`, [one case of which](https://docs.rs/sentry/latest/sentry/struct.ClientOptions.html#impl-From%3C(T%2C%20ClientOptions)%3E-for-ClientOptions) is `(T: IntoDsn, ClientOptions)` , and there is [`IntoDsn<&'a str>`](https://docs.rs/sentry/latest/sentry/trait.IntoDsn.html#impl-IntoDsn-for-%26%27a%20str).